platform/ethereum/client.go

```diff
@@ -72,14 +73,20 @@ func (c *Client) CurrentBlockNumber() (num int64, err error) {
 	path := fmt.Sprintf("%s/node_info", c.BaseURL)
 	res, err := http.Get(path)
 	if err != nil {
-		return num, err
+		return num, errors.E(err, errors.TypePlatformRequest, errors.Params{
+			"coin":   "Ethereum",
```

Comment:
can we handle this error on the higher level? so it would log all errors for all providers

Reply:
We don't go to handle now. We can use this package to put more information inside the error. This error doesn't go to be logged, now. We are going to handle this in higher live with the logger package